h2o-algos/src/main/java/hex/tree/Score.java

```diff
@@ -17,19 +19,33 @@
   final boolean _oob;           // Computed on OOB
   final Key<Vec> _kresp;        // Response vector key (might be either train or validation)
   final ModelCategory _mcat;    // Model category (Binomial, Regression, etc)
-  ModelMetrics.MetricBuilder _mb;
-//  GainsLift.GainsLiftBuilder _gainsLiftBuilder;
+  ModelMetricsSupervised.MetricBuilderSupervised _mb;
   final boolean _computeGainsLift;
+  final ScoreIncInfo _sii;      // Incremental scoring (on a validation dataset), null indicates full scoring
+  final Frame _preds;           // Prediction cache (typically not too many Vecs => it is not too costly embed the object in MRTask)
```

Comment:
should be Key, no?

Reply:
The frame is are actually not installed in DKV. For this case I think it makes a very little difference whether we serialize a key or the frame.

Reply:
ok